src/agent.ts

```diff
@@ -54,6 +56,33 @@ export class Agent {
     this.connection = connection;
   }
 
+  /**
+   * List all agents in the current project.
+   */
+  public static async list(project: SfProject): Promise<string[]> {
+    const projectDirs = project.getPackageDirectories();
+    const bots: string[] = [];
+
+    for (const pkgDir of projectDirs) {
+      const botsPath = path.join(pkgDir.fullPath, 'main', 'default', 'bots');
```

Comment:
for non-default dirs, they don't need the `main/default`, can just be `myPkg/bots`